[DOCS][8.19.0 & 8.19.1][Known issue] Reports in non-default spaces not retrieved by job/list API (#231058)
elastic/kibana: docs/CHANGELOG.asciidoc

=== Known issues
include::CHANGELOG.asciidoc[tag=known-issue-230275]

// tag::known-issue-2462[]

> .Reports created in alternative Kibana spaces aren't retrieved by the lists or job API.

Author: ```suggestion
.Reports created in non-default Kibana spaces aren't shown in the Reporting UI.
```

[%collapsible]
====
*Details* +

> After creating a report in a non-default {kib} space, the document exists in the `.kibana-reporting` index but can't be retrieved by the jobs or list API.

Author: ```suggestion
After creating a report in a non-default {kib} space, the document exists in the `.kibana-reporting` index but isn't shown in the Reporting UI.
```
